x-pack/test/plugin_functional/plugins/resolver_test/public/applications/resolver_test/index.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import { AppMountContext, AppMountParameters } from 'kibana/public';
+import { I18nProvider } from '@kbn/i18n/react';
+import { IEmbeddable } from 'src/plugins/embeddable/public';
+import { useEffect } from 'react';
+
+/**
+ * This module will be loaded asynchronously to reduce the bundle size of your plugin's main bundle.
+ */
+export function renderApp(
+  ...args: [AppMountContext, AppMountParameters, Promise<IEmbeddable | undefined>]
```

Comment:
Maybe not pass in `AppMountContext` since you don't need it?  Then maybe could make this syntax a little simpler

```
export function renderApp({ element}: AppMountParams, embeddable: Promise<IEmbeddable | undefined>) {
...
}
```
If you are thinking that you will later want to use context and you don't want to worry about changing the order of params, you could take in an object as a single param and add it in later.


```
export function renderApp({ { element }, embeddable }: { params: AppMountParams, embeddable: Promise<IEmbeddable | undefined>}) {
...
}
```


I haven't seen anyone else use this kind of syntax, it throws me off a little, personally not a fan, but take this comment as an optional nit, continue to use if you prefer!

Reply:
I'll fix this up. For some reason I had it in my head that `renderApp` had to follow a certain interface. I realized it didn't, but the I didn't go back and clean this up.